src/components/rubric-levels-mobile.js

```diff
@@ -0,0 +1,1038 @@
+		if (levelClass) {
+			const levels = entity.getSubEntitiesByClass(levelClass);
+			this._levelEntities = levels;
+			this.dispatchEvent(new CustomEvent('d2l-rubric-level-entities', { detail: { levelEntities: levels }, bubbles: true, composed: true }));
+			if (!this.criterionCells) {
+				this.criterionCells = levels;
+			}
+		}
+	}
+
+	_queueAction(entityHref, updateSelected = false) {
+		if (this.readOnly) return;
+		const entityGetter = () => this.assessmentMap[entityHref];
+		this._pendingChanges++;
+		if (this.onSelectOptimistic) {
+			this.onSelectOptimistic(this.assessmentMap[entityHref]);
+		}
+		this.onSelect(entityGetter, () => {
+			this._pendingChanges--;
+			if (updateSelected) {
+				this._selectAssessedLevel(this.criterionCells, this._assessmentEntity);
+			}
+		});
+	}
+
+	_renderGradient() {
+		return html`
+				${this._canScrollLeft(this._scroll) ? html`<div class="gradient-mask left"></div>` : null}
+				${this._canScrollRight(this._scroll, this._width) ? html`<div class="gradient-mask right"></div>` : null}
+		`;
+	}
+
+	_renderLevel(level, index) {
+		return html`
+			<div
+				role="radio"
+				name="${this._getCriterionId()}"
+				id="${this._getLevelId(index)}"
+				class="${this._getLevelClassName(index, this._selected, this._hovered, this._focused, this._animating)}"
+				dataIndex="${index}"
+				cellHref="${this._getCriterionCellHref(this.criterionCells, index)}"
+				?aria-checked="${this._getCriterionCellAriaChecked(index, this._selected)}"
+				tabindex="${this._getCriterionCellTabIndex(index, this._focused)}"
+				@mouseover="${this._handleMouseOver}"
+				@mouseout="${this._handleMouseOut}"
+				@mousedown="${this._handleMouseDown}"
+				@focus="${this._handleFocus}"
+				@blur="${this._handleBlur}"
+				@keydown="${this._handleKey}"
+			>
+				<d2l-offscreen>
+					${this._getLevelText(index, this.criterionCells, this._levelEntities)}
+				</d2l-offscreen>
+			</div>
+		`;
+	}
+
+	_renderThumb() {
+		if (this._selected < 0) {
+			return;
+		}
+		const thumbClasses = classMap({
+			'level-thumb': true,
+			focused: this._focused >= 0 && this._focused === this._selected,
+			dragging: this._hasDragStarted
+		});
+
+		const levelWidth = this._getLevelWidth();
+		if (!this._levelEntities || this._selected === -1 || !levelWidth) {
+			return 'display: none;';
+		}
+		const thumbWidth = levelWidth - 1;
+		const thumbX = levelWidth * this._selected;
+		const thumbStyles = styleMap({
+			width: `${thumbWidth}px`,
+			marginLeft: `calc(${thumbX}px)`
+		});
+
+		return html`
+			<span
+				class="${thumbClasses}"
+				style="${thumbStyles}"
+				@mousedown="${this._handleThumbMouseDown}"
+				@transitionend="${this._handleThumbTransitionEnd}"
+			>
+				<d2l-icon class="check-icon" icon="d2l-tier1:check"></d2l-icon>
+			</span>
+		`;
+	}
+
+	_say(text) {
+		const statusWrap = this.shadowRoot.querySelector('.criterion-status');
+		const status = statusWrap && statusWrap.querySelector('p');
```

Comment:
```suggestion
		const status = this.shadowRoot.querySelector('.criterion-status p');
```
I actually added this before I learned about `@brightspace-ui/core/helpers/announce`. So we could probably scrap this whole function and replace it with the core helper. Only possible downside to discuss is whether inline rubrics can be opened in `d2l-dialog` which would still break aria-live afaik.

Reply:
I'm pretty sure it can't be opened in a dialog as far as I'm aware... 🤔 
I'll make your suggested change and we can circle back to this if other people are confident this can't be in a dialog as well.